src/core/dev/hw_queue_tx.cpp

```diff
@@ -1325,7 +1320,7 @@ inline void hw_queue_tx::tls_post_progress_params_wqe(xlio_ti *ti, uint32_t tis_
     uint8_t opmod =
         is_tx ? MLX5_OPC_MOD_TLS_TIS_PROGRESS_PARAMS : MLX5_OPC_MOD_TLS_TIR_PROGRESS_PARAMS;
 
-    memset(wqe, 0, sizeof(*wqe));
+    memset(wqe, 0, sizeof(mlx5_eth_wqe));
```

Comment:
Replace `sizeof(mlx5_eth_wqe)` with either `WQEBB` or `max<size_t>(WQEBB, sizeof(mlx5_set_tls_progress_params_wqe))`. See comment above. Repeat the same for all similar places.

Reply:
Done